source/pervasive/map.rs

```diff
@@ -107,6 +107,13 @@ impl<K, V> Map<K, V> {
         self.dom().contains(k)
     }
 
+    /// Returns true if the value `v` is in the map of `self`.
+
+    #[verifier(inline)]
+    pub open spec fn contains_value(self, v: V) -> bool {
+        exists|i: K| #[trigger] self.dom().contains(i) && self.index(i) == val
```

Comment:
Sorry I didn't spot this earlier, can you use `self[i]` instead?

Reply:
Done!